cli/src/cast_opts.rs

```diff
@@ -126,6 +132,14 @@ pub enum Subcommands {
         #[structopt(flatten)]
         eth: EthereumOpts,
     },
+    #[structopt(name = "abi-decode")]
+    #[structopt(about = "Decode ABI-encoded hex data")]
+    abiDecode {
+        #[structopt(help = "the hex data you want to decode")]
```

Comment:
This should be on the `calldata` field below. 

This field should instead be called `sig` and its help should be something like:

```
#[structopt(help = "the function signature you want to decode, in the format `<name>(<in-types>)(<out-types>)`")]
```